Scenes/TestScenes/Dialogue/TestDialogue.cs

```diff
@@ -1,21 +1,28 @@
 using Godot;
 using untitledplantgame.Common.GameStates;
-using untitledplantgame.Dialogue;
+using DialogueResourceObject = untitledplantgame.Dialogue.Models.DialogueResourceObject;
+using DialogueSystem = untitledplantgame.Dialogue.DialogueSystem;
+
+namespace untitledplantgame.Scenes.TestScenes.Dialogue;
 
 public partial class TestDialogue : Node2D
 {
-	[Export] private DialogueResourceObject _exampleDialogue;
-	[Export] private DialogueResourceObject _followupDialogue;
+	[Export]
+	private DialogueResourceObject _exampleDialogue;
+
+	[Export]
+	private DialogueResourceObject _followupDialogue;
+
 	public override void _Ready()
 	{
 		// Prepare
-			
-			
+
+
 		GD.Print($"Current game state: {GameStateMachine.Instance.CurrentState}");
 		GD.Print("Trigger dialogue.");
-		
+
 		DialogueSystem.Instance.StartDialog(_exampleDialogue._dialogueId);
-		
+
 		GD.Print($"Current game state: {GameStateMachine.Instance.CurrentState}");
```

Comment:
logger

Reply:
but also i think code specifically only for the tests are okay if not clean with print

Reply:
prints for debugging the code to make it work as it should is totally fine. But when it stays, it should be commented or changed to logger